convert2rhel/subscription.py

```diff
@@ -560,6 +560,20 @@ def install_rhel_subscription_manager(pkgs_to_install, pkgs_to_upgrade=None):
     backup.backup_control.push(installed_pkg_set)
 
 
+def is_sca_enabled():
+    """
+    Check if Simple Content Access has been enabled for this system.
+    :returns: True if Simple Content Access is enabled.
+    :rtype: bool
+    """
+    # check if SCA is enabled
+    output, _ = utils.run_subprocess(["subscription-manager", "status"], print_output=False)
+    if "content access mode is set to simple content access." in output.lower():
+        loggerinst.info("Simple Content Access is enabled, skipping subscription attachment")
```

Comment:
I would not put in the `skipping subscription attachment` here in case we ever re-use this function somewhere else, it might give a confusing message, I would put the log back where it was if anything